crates/matrix-sdk-ui/src/timeline/event_item/content/msg_like.rs

```diff
@@ -36,17 +36,31 @@ pub enum MsgLikeKind {
     UnableToDecrypt(EncryptedMessage),
 }
 
+#[derive(Clone, Debug)]
+pub struct ThreadSummary {
+    pub latest_event_details: TimelineDetails<Box<ThreadSummaryLatestEvent>>,
```

Comment:
Let's name this `latest_event` (details refers to `TimelineDetails` and is rather an implementation detail (heh))